test/Microsoft.AspNetCore.Server.KestrelTests/TestConnection.cs

```diff
@@ -125,5 +125,28 @@ public void Dispose()
             var text = new string(ch, 0, count);
             Assert.Equal("", text);
         }
+
+        public static Socket CreateConnectedLoopbackSocket(int port, AddressFamily addressFamily = AddressFamily.InterNetwork)
```

Comment:
I don't see any calls overriding the optional `addressFamily` parameter. Move to method body?